src/cursor/cursor.ts

```diff
@@ -875,10 +944,10 @@ export class Cursor<
         }
       });
 
-      return this.pipe(stream);
+      return this.stream().pipe(stream);
```

Comment:
```suggestion
      return this.stream(options).pipe(stream);
```

Reply:
Actually I had to revert this, passing in the options causes it to get double-transformed. Once in `read()` and once by the `Transform` stream.

Reply:
ok sounds good. Yes, I think we should likely remove the `transformStream` method, it was an attempt at providing a better `stream` without breaking compatibility. Can you make a new ticket to remove that method? We'll need to deprecate it in 3.6+ and ensure we have documentation around the removal